pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "legume-gme-tocha"
+version = "0.1.0"
+description = "Guided-mode expansion for 2.5D photonic crystals."
+authors = ["Peter Koval <koval.peter@gmail.com>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "legume"}]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+scipy = "^1.9.3"
+matplotlib = "^3.6.2"
+autograd = "^1.5"
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^7.2.0"
+bumpversion = "^0.6.0"
+wheel = "^0.38.4"
+
+[build-system]
+requires = ["poetry-core"]
```

Comment:
Specifications: Since we are generating the poetry.lock with poetry 1.3, this should be `poetry-core>=1.4.0`.

Reply:
Sorry, if we want to keep compatibility with Poetry 1.2.2, this should be `poetry-core>=1.3.2`.